en/components/general/update_guide.md

```diff
@@ -44,6 +44,17 @@ Unfortunately not all changes can be automatically updated. Changes bellow are s
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+### From 8.0.x to 8.1.x
```

Comment:
_You can extend to the following with better wording (taken from the changelog and new samples fro paginator):_

`igx-paginator` component is introduced and is used in the grids as well is used in the grid components
Have in mind that if you have set the `paginationTemplate`, you may have to modify your CSS to display the pagination correctly. This is due to the fact that some CSS rules (like display: flex) were removed and now you might need to add them manually.
The style should be something similar to:

```html
<igx-grid #grid [data]="data" [paging]="true" [perPage]="10" [paginationTemplate]="pager">
</igx-grid>
<ng-template #pager>
    <div class="pagination-container"></div>
</ng-template>
```

```css
.pagination-container {
    display: flex;
    justify-content: center;
    align-items: center;
}